mizar/daemon/interface_service.py

```diff
@@ -144,6 +135,14 @@ def _ConsumeInterfaces(self, pod_name, cni_params):
         logger.info("Consumed {}".format(interfaces))
         return interfaces
 
+    def _DeleteInterfaces(self, pod_name, cni_params):
+        with self.interfaces_lock:
+            interfaces = self.interfaces.get(pod_name, [])
+            for interface in interfaces:
+                if interface.interface_type == InterfaceType.veth:
+                    self._DeleteVethInterface(interface, cni_params)
+                    logger.info("Removed {}".format(interface))
```

Comment:
I believe this is problematic. This loop will delete all the interfaces for the Pod. With one interface scenario, that's fine. With multiple interfaces, this function must only delete the interface requested by the CNI.